frontend/src/app/HeaderTools.tsx

```diff
@@ -175,16 +177,24 @@ const HeaderTools: React.FC<HeaderToolsProps> = ({ onNotificationsClick }) => {
         )}
         <ToolbarItem>
           <Dropdown
-            isPlain
-            position={DropdownPosition.right}
-            toggle={
-              <DropdownToggle id="user-menu-toggle" onToggle={() => setUserMenuOpen(!userMenuOpen)}>
+            popperProps={{ position: 'right' }}
+            onOpenChange={(isOpen) => setUserMenuOpen(isOpen)}
+            toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+              <MenuToggle
+                variant="plainText"
+                aria-label="User items"
```

Comment:
This would open the user menu.
```suggestion
                aria-label="User menu"
```